framework/web/Request.php

```diff
@@ -1355,7 +1347,7 @@ protected function loadCsrfToken()
      */
     protected function generateCsrfToken()
     {
-        $token = Yii::$app->getSecurity()->generateRandomString();
+        $token = Yii::$app->getSecurity()->generateRandomKey();
```

Comment:
Not in all browsers. There are various problems with non-ASCII: http://stackoverflow.com/questions/1969232/allowed-characters-in-cookies

Also see RFC: https://tools.ietf.org/html/rfc6265#page-8